default.xml

```diff
@@ -30,15 +30,15 @@
   <project name="projects_libs.git" path="projects/projects_libs" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
   <project name="pruner.git" path="tools/pruner" revision="6df378e8d91856d204e4d024796b91c6802d0501" upstream="master" dest-branch="master"/>
   <project name="rumprun.git" path="tools/rumprun" revision="04464ab931c28ca1c78e807741d82a4792660183" upstream="master" dest-branch="master"/>
-  <project name="seL4.git" path="kernel" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
+  <project name="seL4.git" path="kernel" remote="seL4" revision="ae7798e3193a223de5c841bdf6ad319937b3403b" upstream="master" dest-branch="master"/>
```

Comment:
Questioning the equivalence of this change offline ... will update

Reply:
This url shows the diffs between the repos, I believe: https://github.com/sel4devkit/seL4/compare/master..seL4:seL4:ae7798e3193a223de5c841bdf6ad319937b3403b
where green pluses signify lines that are in the Foundation repo, not our local one, with this ordering (the url can be "reversed" (with different / and : delimiters) to show the diff the other way round). Note that .. vs ... diffs (the url supports both types) are explained [here](https://stackoverflow.com/questions/7251477/what-are-the-differences-between-double-dot-and-triple-dot-in-git-dif/7256391#7256391).

Reply:
The differences are not considered to be significant, so the ae7798e revision is acceptable. (Will still do a confirmatory build with the new manifest for good measure - TBD)